### PR TITLE
neofs-cli: Check the sufficiency of the number of nodes in the selector for replicas

### LINKED ISSUE #2038: Container placement policy allows wrong policy with REP > SELECT * CBF
## Expected Behavior
```
could not build container nodes based on given placement policy, use --force option to skip this check: REP < SELECT * CBF
```

## Current Behavior
It's allowed a policy like "REP 4 IN X CBF 1 SELECT 2" that causes error on the put object command 

## Steps to Reproduce (for bugs)
1. create container with REP > SELECT * CBF
```
# neofs-cli container create -r node1:8080 -w /etc/neofs/storage/wallet.json --policy "REP 4 IN X CBF 1 SELECT 2 FROM * AS X" --await --basic-acl public-read-write
Enter password >
container ID: 38LJ8VNvH3BTWB4nDVokLZqVZDpwB9yMV4cFbyRDEayr
awaiting...
container has been persisted on sidechain
```
2. try to put an object in the container - ERROR
```
# neofs-cli object put -r node1:8080 -w /etc/neofs/storage/wallet.json  --file /home/service/.profile --cid 38LJ8VNvH3BTWB4nDVokLZqVZDpwB9yMV4cFbyRDEayr
Enter password >
 807 / 807 [==========================================================================================================================================================] 100.00% 0s
rpc error: client failure: status: code = 1024 message = incomplete object PUT by placement
```

## Your Environment
Sbercloud 4 nodes
```
# neofs-node --version
NeoFS Storage node
Version: v0.34.0-26-gc38ad2d3
GoVersion: go1.18.4
root@az:/home/service# neofs-cli --version
NeoFS CLI
Version: v0.34.0-26-gc38ad2d3
GoVersion: go1.18.4
```

### PR DESCRIPTION
**Pull request by acid-ant**
Perform this check on container creation.

Close #2038

Signed-off-by: Anton Nikiforov <an.nikiforov@yadro.com>

### CODE REVIEW

**Comment by sami-nspcc** [comment]
I have recieved a notification of a new pull request. I am starting the build of images and binaries for further testing. Build number is 3742

File: cmd/neofs-cli/modules/container/create.go [hunk 1]
```diff
@@ -50,9 +50,15 @@ It will be stored in sidechain when inner ring will accepts it.`,
...
 				"use --force option to skip this check: %w", err)
 
-			_, err = resmap.NetMap().ContainerNodes(*placementPolicy, nil)
+			nodesByRep, err := resmap.NetMap().ContainerNodes(*placementPolicy, nil)
 			common.ExitOnErr(cmd, "could not build container nodes based on given placement policy, "+
 				"use --force option to skip this check: %w", err)
+
+			for i, nodes := range nodesByRep {
+				if placementPolicy.ReplicaNumberByIndex(i) > uint32(len(nodes)) {
+					common.ExitOnErr(cmd, "", fmt.Errorf("not enough nodes in selector for replicas"))
```
Comment (fyrchik) [inline cmd/neofs-cli/modules/container/create.go:59]: Can we also give some numbers in the error message?

Comment (acid-ant) [reply cmd/neofs-cli/modules/container/create.go:59]: How bout this: `the number of nodes '%d' is not enough for the number of replicas '%d'`

**Comment by sami-nspcc** [comment]
I have recieved a notification of a new pull request. I am starting the build of images and binaries for further testing. Build number is 3743

File: cmd/neofs-cli/modules/container/create.go [hunk 2]
```diff
@@ -50,9 +50,19 @@ It will be stored in sidechain when inner ring will accepts it.`,
...
 
-			_, err = resmap.NetMap().ContainerNodes(*placementPolicy, nil)
+			nodesByRep, err := resmap.NetMap().ContainerNodes(*placementPolicy, nil)
 			common.ExitOnErr(cmd, "could not build container nodes based on given placement policy, "+
 				"use --force option to skip this check: %w", err)
+
+			for i, nodes := range nodesByRep {
+				if placementPolicy.ReplicaNumberByIndex(i) > uint32(len(nodes)) {
+					common.ExitOnErr(cmd, "", fmt.Errorf(
+						"the number of nodes '%d' in selector is not enough for the number of replicas '%d'",
```
Comment (carpawell) [inline cmd/neofs-cli/modules/container/create.go:60]: how about `--force` flag mentioning in the err msg?

Comment (acid-ant) [reply cmd/neofs-cli/modules/container/create.go]: Thanks! Added.

**Comment by sami-nspcc** [comment]
I have recieved a notification of a new pull request. I am starting the build of images and binaries for further testing. Build number is 3745

**Review by carpawell — APPROVED** [review]

**Comment by sami-nspcc** [comment]
I have recieved a notification of a new pull request. I am starting the build of images and binaries for further testing. Build number is 3750

**Review by fyrchik — APPROVED** [review]

**Comment by sami-nspcc** [comment]
I have recieved a notification of a new pull request. I am starting the build of images and binaries for further testing. Build number is 3758

**Pull request merged**